libnmstate/nm/team.py

```diff
@@ -25,9 +25,18 @@
 from libnmstate.schema import Team
 
 
+CAPABILITY = "team"
 TEAMD_JSON_DEVICE = "device"
 
 
+def has_team_capability():
+    nm_client = nmclient.client()
+    if nmclient.NM.Capability.TEAM in nm_client.get_capabilities():
```

Comment:
you can simplify this by doing `return nmclient.NM.Capability.TEAM in nm_client.get_capabilities()`

Reply:
Thanks! Done :-)